docs/kpod-stats.1.md

```diff
@@ -0,0 +1,37 @@
+% kpod(1) kpod-stats - Display a live stream of 1 or more containers' resource usage statistics
+% Ryan Cole
+# kpod-stats "1" "July 2017" "kpod"
+
+## NAME
+kpod-stats - Display a live stream of 1 or more containers' resource usage statistics
+
+## SYNOPSIS
+**kpod** **stats** [*options* [...]] [container]
+
+## DESCRIPTION
+Display a live stream of one or more containers' resource usage statistics
+
+## OPTIONS
+
+**--all, -a**
+
+Show all containers.  Only running containers are shown by default
+
+**--no-stream**
+
+Disable streaming stats and only pull the first result, default setting is false
+
+**--format="TEMPLATE"**
```

Comment:
It does, but I believe there are other pull requests for that which haven't been merged.  If they get merged first, I'll fix this.  Otherwise I'll submit a second PR